pkgs/development/libraries/gts/default.nix

```diff
@@ -10,11 +10,14 @@ stdenv.mkDerivation rec {
     sha256 = "07mqx09jxh8cv9753y2d2jsv7wp8vjmrd7zcfpbrddz3wc9kx705";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ glib gettext ];
+  nativeBuildInputs = [ pkgconfig autoreconfHook ];
+  buildInputs = [ gettext ];
+  propagatedBuildInputs = [ glib ];
 
   doCheck = false; # fails with "permission denied"
 
+  outputs = [ "bin" "dev" "out" ];
```

Comment:
we typically add outputs right after `pname` & `version`, so it is immediately obvious what is built. See https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887

Reply:
ok, fixed